src/Sulu/Bundle/MediaBundle/Resources/config/doctrine/FormatOptions.orm.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+
+    <entity name="Sulu\Bundle\MediaBundle\Entity\FormatOptions"
```

Comment:
Didn't we agree that these options could be part of the `FileVersion`? Or what was your intention to put that into an own table/entity?